source/Halibut/Transport/SecureClient.cs

```diff
@@ -68,7 +66,7 @@ public void ExecuteTransaction(ExchangeAction protocolHandler, CancellationToken
                     IConnection connection = null;
                     try
                     {
-                        connection = connectionManager.AcquireConnection(protocolBuilder, new TcpConnectionFactory(clientCertificate, asyncHalibutFeature, halibutTimeoutsAndLimits), ServiceEndpoint, log, cancellationToken);
+                        connection = connectionManager.AcquireConnection(protocolBuilder, tcpConnectionFactory, ServiceEndpoint, log, cancellationToken);
```

Comment:
We are starting to reduce our "toggle off has no changes to sync halibut" approach. Our we consciously doing this now? Does it start to introduce more risk?